lib/buildr_plus/features/less.rb

```diff
@@ -17,17 +17,21 @@
     def default_less_path
       'src/main/webapp/less'
     end
+
+    attr_accessor :options
```

Comment:
These are global "default" options so they should probably be named default_options.